composer.json

```diff
@@ -5,13 +5,14 @@
   "prefer-stable": true,
   "license": "MIT",
   "require": {
-    "byjg/webrequest": "4.9.*",
-    "php": ">=7.4",
-    "ext-json": "*"
+    "php": ">=8.1",
```

Comment:
This constraint also allows version 8.4, but this version of PHP is not covered by the GitHub workflow.

Reply:
I am going to upgrade the project https://github.com/byjg/docker-php to support PHP 8.4 since `alpine:edge` has the 8.4 beta already :)

Reply:
The problem is different: this packages can be installed with version 8.4 of PHP but it is not tested with that version. This can therefore lead to problems on the user side, as PHP does not follow [semantic versioning](https://semver.org).
This would help:

```json
"php": "~8.1.0 || ~8.2.0 || ~8.3.0",
```

https://getcomposer.org/doc/articles/versions.md

Reply:
Gotcha. I'll add this.